### ISSUE TITLE
Update axondeepseg help

### ISSUE THREAD

**Issue by alzaia** [neuropoly/axondeepseg#51]
Change input data flag from '-p' to '-i'.

**Comment by jcohenadad** [comment]
also:
- explicit what models to choose from with flag t
- give possible values for -v
- show default model
- fix category of arguments. currently, -t is in "optional" although it is mandatory. Same for -f.
- give ranges of values for -o. Currently the user has no clue what values to use and what the value means. 0? 100029303930? 33?
~~~
axondeepseg -t TEM -p test/ -s 0.02387 -h
usage: axondeepseg [-h] -t TYPE -p PATH [PATH ...] [-m MODEL] [-s SIZEPIXEL]
                   [-v VERBOSE] [-o OVERLAP]

optional arguments:
  -h, --help            show this help message and exit
  -t TYPE, --type TYPE  Choose the type of acquisition you want to segment.
  -p PATH [PATH ...], --path PATH [PATH ...]
                        Folder where the acquisition folder are located
  -m MODEL, --model MODEL
                        Folder where the model is located
  -s SIZEPIXEL, --sizepixel SIZEPIXEL
                        Pixel size in micrometer to use for the segmentation.
  -v VERBOSE, --verbose VERBOSE
                        Verbosity level
  -o OVERLAP, --overlap OVERLAP
                        Overlap value when doing the segmentation. The higher
                        thevalue, the longer it will take to segment the whole
                        image
~~~

**Comment by jcohenadad** [comment]
this issue should be address in a new dedicated PR. See here: https://gist.github.com/mikepea/863f63d6e37281e329f8#keep-it-small

**Comment by alzaia** [comment]
This was resolved with PR #52
